meta-resin-common/recipes-support/resin-device-progress/resin-device-progress/resin-device-progress

```diff
@@ -67,9 +67,9 @@ if [ `jq ".deviceId | length" $CONFIG_PATH` -eq 0 ]; then
 fi
 
 # If the user api key exists we use it instead of the deviceApiKey as it means we haven't done the key exchange yet
-read device_api_key device_id <<<$(jq -r '.apiKey // .deviceApiKey,.deviceId' $CONFIG_PATH)
+_device_api_key=${PROVISIONING_API_KEY:-DEVICE_API_KEY}
```

Comment:
Shouldn't this just fail if there's no device apikey? After all this should only run after registration.